cmd/ingress-operator/start.go

```diff
@@ -65,76 +73,66 @@ func NewStartCommand() *cobra.Command {
 		},
 	}
 
-	cmd.Flags().StringVarP(&options.ShutdownFile, "shutdown-file", "s", defaultTrustedCABundle, "file which when modified should trigger a shutdown. If empty, shutdown triggering is disabled.")
+	cmd.Flags().StringVarP(&options.ShutdownFile, "shutdown-file", "s", defaultTrustedCABundle, "if provided, shut down the operator when this file changes")
+	cmd.Flags().StringVarP(&options.MetricsListenAddr, "metrics-listen-addr", "", ":6000", "metrics endpoint listen address")
```

Comment:
`6000` → `60000`.

Reply:
What about the metrics listening by container `kube-rbac-proxy` on port 9393? Will it be replaced by this?

Reply:
Kube-rbac-proxy listens on port 9393 and forwards to 127.0.0.1:60000: https://github.com/openshift/cluster-ingress-operator/blob/a11c8f1d9a7e4cc2334e959700c1257015f5cf23/manifests/02-deployment.yaml#L61-L72

Reply:
so it seems if we change the port by CLI options, we also need to update `--upstream` in the deployment yaml file?

Reply:
Right.  Prior this change, the port was hard-coded to 60000:  https://github.com/openshift/cluster-ingress-operator/blob/a11c8f1d9a7e4cc2334e959700c1257015f5cf23/cmd/ingress-operator/start.go#L74

The deployment yaml file references port 60000, so we should use 60000 as the default value for the command-line option (or update the dpeloyment yaml to use the new port number).

Reply:
Fixed